<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBarView</class>
 <widget class="QWidget" name="ToolBarView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>170</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pencilButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="myResources.qrc">
     <normaloff>:/icons/pencil-svg.svg</normaloff>:/icons/pencil-svg.svg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="eraserButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="myResources.qrc">
     <normaloff>:/icons/eraser-svg.svg</normaloff>:/icons/eraser-svg.svg</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>128</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="colorGrid">
    <item row="0" column="0">
     <widget class="QToolButton" name="color1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="color2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="color3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="color4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="color5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QToolButton" name="color6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="selectColor">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Colors</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="myResources.qrc"/>
 </resources>
 <connections/>
</ui>
